src/components/section/section.jsx

```diff
@@ -0,0 +1,26 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import styles from './section.css';
+
+export default function Section(props) {
+  return (
+    <section className={props.className || styles.section}>
+      <h2 className={styles.subheading}>{props.title}</h2>
+      {props.description && <p className={styles.description}>{props.description}</p>}
+      {props.children}
+    </section>
+  );
+}
+
+Section.propTypes = {
+  children: PropTypes.node,
+  className: PropTypes.string,
```

Comment:
What is a use-case for overriding the styling on a section?

Reply:
If you want different behavior for the child rows, I imagine. There are a few different ways to slice that problem, with one of them being an override for the styles. Is that something we want to discourage? Are we confident that we can achieve all possible design specs without such an override?

Reply:
>  Is that something we want to discourage?

Hmmm... That is a good question: we are already implementing the design spec for a section component and it is pretty simple. Perhaps it is a good idea to avoid letting other change the styling for the section component.

> Are we confident that we can achieve all possible design specs without such an override?

I think a reasonable guess to customizing the styling for the section component would be to customize the rows of the sections. However, the rows are the children of the section which means any dev can just pass down their own fully styled rows if they need overrides. Those rows can even be composing from the future section-row component. 

I think overall, I'm still not sure what is the best outcome. I will let you decide now that we talked about implementing a SectionRow component.